cs/build/internal/Common.Internal.props

```diff
@@ -9,11 +9,17 @@
     <BOND_INCLUDE_PATH Condition="'$(BOND_INCLUDE_PATH)' == ''">$(MSBuildThisFileDirectory)..\..\..\idl</BOND_INCLUDE_PATH>
   </PropertyGroup>
   <Import Project="..\nuget\Common.props" />
-  <PropertyGroup Condition="'$(BuildFramework)' == ''">
+  <PropertyGroup Condition="'$(BuildFramework)' == '' and '$(UsingMicrosoftNETSdk)' != 'true'">
     <BuildFramework>net45</BuildFramework>
   </PropertyGroup>
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true'">
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdk)' == 'true'">
+    <!-- We need to disable automatic AssemblyInfo generation to avoid duplication with our AssemblyInfo file -->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
We can probably always disable this: if so, it can be moved into the main property group with `FileAlignment` and all that jazz.